common/src/main/java/com/datatorrent/common/util/PubSubWebSocketClient.java

```diff
@@ -289,11 +298,11 @@ public void assertUsable() throws IOException
     Throwable t = throwable.get();
     if (t instanceof IOException) {
       throw (IOException)t;
-    }
-    else {
+    } else {
       DTThrowable.rethrow(t);
```

Comment:
Not sure if we should use this opportunity to remove all DTThrowable.rethrow() calls.